demo/gtk-demo/builder.lisp

```diff
@@ -0,0 +1,44 @@
+;;;; Glade and GtkBuilder
+
+(in-package #:gtk-demo)
+
+(defvar *xml*
+  "<?xml version=\"1.0\" encoding=\"UTF-8\"?>
```

Comment:
This could use one of the better string reader macros. I don't want to put it in a separate file though.

Reply:
Note that using a separate file is already done is some demos (e.g. `demo/gtk-example/dialog.ui`).